Mathlib/Order/Filter/Pointwise.lean

```diff
@@ -73,77 +73,77 @@ section One
 
 variable [One α] {f : Filter α} {s : Set α}
 
-/-- `1 : filter α` is defined as the filter of sets containing `1 : α` in locale `pointwise`. -/
+/-- `1 : Filter α` is defined as the filter of sets containing `1 : α` in locale `Pointwise`. -/
 @[to_additive
-      "`0 : filter α` is defined as the filter of sets containing `0 : α` in locale\n`pointwise`."]
-protected def hasOne : One (Filter α) :=
+      "`0 : Filter α` is defined as the filter of sets containing `0 : α` in locale `Pointwise`."]
+protected def instOne : One (Filter α) :=
   ⟨pure 1⟩
-#align filter.has_one Filter.hasOne
-#align filter.has_zero Filter.hasZero
+#align filter.has_one Filter.instOne
+#align filter.has_zero Filter.instZero
 
-scoped[Pointwise] attribute [instance] Filter.hasOne Filter.hasZero
+scoped[Pointwise] attribute [instance] Filter.instOne Filter.instZero
 
-@[simp, to_additive]
+@[to_additive (attr := simp)]
 theorem mem_one : s ∈ (1 : Filter α) ↔ (1 : α) ∈ s :=
   mem_pure
 #align filter.mem_one Filter.mem_one
 #align filter.mem_zero Filter.mem_zero
 
 @[to_additive]
 theorem one_mem_one : (1 : Set α) ∈ (1 : Filter α) :=
-  mem_pure.2 one_mem_one
+  mem_pure.2 Set.one_mem_one
 #align filter.one_mem_one Filter.one_mem_one
 #align filter.zero_mem_zero Filter.zero_mem_zero
 
-@[simp, to_additive]
+@[to_additive (attr := simp)]
 theorem pure_one : pure 1 = (1 : Filter α) :=
   rfl
 #align filter.pure_one Filter.pure_one
 #align filter.pure_zero Filter.pure_zero
 
-@[simp, to_additive]
+@[to_additive (attr := simp)]
 theorem principal_one : 𝓟 1 = (1 : Filter α) :=
   principal_singleton _
 #align filter.principal_one Filter.principal_one
 #align filter.principal_zero Filter.principal_zero
 
 @[to_additive]
-theorem one_neBot : (1 : Filter α).ne_bot :=
+theorem one_neBot : (1 : Filter α).NeBot :=
```

Comment:
This should be a (scoped) instance. If this breaks compatibility with mathlib 3, then please either backport the change, or add a `todo` instead of making it an instance now.